crates/rpc-types/src/receipt.rs

```diff
@@ -48,19 +48,25 @@ pub struct OptimismTransactionReceiptFields {
     /// L1 fee for the transaction
     #[serde(default, skip_serializing_if = "Option::is_none", with = "alloy_serde::quantity::opt")]
     pub l1_fee: Option<u128>,
-    /// L1 fee scalar for the transaction
-    #[serde(default, skip_serializing_if = "Option::is_none", with = "l1_fee_scalar_serde")]
-    pub l1_fee_scalar: Option<f64>,
+    /// L1 base fee scalar for the transaction
+    #[serde(default, skip_serializing_if = "Option::is_none", with = "l1_base_fee_scalar_serde")]
+    pub l1_base_fee_scalar: Option<f64>,
     /// L1 gas price for the transaction
     #[serde(default, skip_serializing_if = "Option::is_none", with = "alloy_serde::quantity::opt")]
     pub l1_gas_price: Option<u128>,
     /// L1 gas used for the transaction
     #[serde(default, skip_serializing_if = "Option::is_none", with = "alloy_serde::quantity::opt")]
     pub l1_gas_used: Option<u128>,
+    /// L1 blob base fee for transaction
+    #[serde(default, skip_serializing_if = "Option::is_none", with = "alloy_serde::quantity::opt")]
+    pub l1_blob_base_fee: Option<u128>,
+    /// L1 blob base fee scalar for transaction
+    #[serde(default, skip_serializing_if = "Option::is_none", with = "l1_base_fee_scalar_serde")]
+    pub l1_blob_base_fee_scalar: Option<f64>,
```

Comment:
this is not f64

Reply:
will branch that out to separate pr. for now, with using `AnyTransactionReceipt` in reth, we're should be good with that change. however soon with https://github.com/paradigmxyz/reth/issues/10452, it will be a problem that `OpTransactionReceipt` doesn't store any of the op fields, except for deposit via deposit envelope. made issue of this discovery https://github.com/alloy-rs/op-alloy/issues/54 and leaving this pr open as draft.